chef/cookbooks/ceph/recipes/role_ceph_osd.rb

```diff
@@ -16,4 +16,5 @@
 
 if CrowbarRoleRecipe.node_state_valid_for_role?(node, "ceph", "ceph-osd")
   include_recipe "ceph::osd"
+  include_recipe "ceph::monitor_monasca"
```

Comment:
do we need it on the osd nodes? or just mon nodes?

Reply:
mon and OSD - definitely not radosgw, calamari and mds (fixed)